firebase-ai-friendly-meals/android/app/src/main/java/com/google/firebase/example/friendlymeals/ui/home/HomeScreen.kt

```diff
@@ -127,23 +169,54 @@ fun IngredientsBox(
             .padding(16.dp)
     ) {
         Column(modifier = Modifier.fillMaxWidth()) {
-            TextField(
-                value = ingredients,
-                onValueChange = { ingredients = it },
-                modifier = Modifier
-                    .fillMaxWidth().height(128.dp),
-                shape = RoundedCornerShape(24.dp),
-                colors = TextFieldDefaults.colors(
-                    focusedIndicatorColor = Color.Transparent,
-                    unfocusedIndicatorColor = Color.Transparent,
-                    disabledIndicatorColor = Color.Transparent,
-                    focusedContainerColor = LightFirebaseYellow,
-                    unfocusedContainerColor = LightFirebaseYellow
-                ),
-                placeholder = {
-                    Text(text = stringResource(R.string.ingredients_hint))
+            Box {
+                TextField(
+                    value = viewState.ingredients,
+                    onValueChange = onIngredientsUpdated,
+                    modifier = Modifier
+                        .fillMaxWidth().height(128.dp),
+                    shape = RoundedCornerShape(24.dp),
+                    colors = TextFieldDefaults.colors(
+                        focusedIndicatorColor = Color.Transparent,
+                        unfocusedIndicatorColor = Color.Transparent,
+                        disabledIndicatorColor = Color.Transparent,
+                        focusedContainerColor = LightFirebaseYellow,
+                        unfocusedContainerColor = LightFirebaseYellow
```

Comment:
This creates poor contrast between the text and the container when Dark Mode is on. We may need to find an alternative dark background color to use in dark mode.

(same for the disabled button)

![friendlymeals_darkmode](https://github.com/user-attachments/assets/727e0984-0315-4399-adb0-9761f9d862b8)